users/stanrc85/rgblight_layers.c

```diff
@@ -34,6 +35,22 @@ const rgblight_segment_t* const PROGMEM my_rgb_layers[] = RGBLIGHT_LAYERS_LIST(
 void keyboard_post_init_user(void) {
     // Enable the LED layers
     rgblight_layers = my_rgb_layers;
+    // Boot up "fanfare"
```

Comment:
Nice!

However, it may be better to use `matrix_scan_user` to do this, and software times.  Otherwise, this will block the initialization until it's done.

Something like: https://github.com/qmk/qmk_firmware/blob/c9eaf1ac2ba0e9114343815432de524062f4c94f/users/drashna/rgb_stuff.c#L132-L150

Reply:
Thanks for tip, I put updated version in its own startup_fanfare.c to keep it cleaner.  I kept the `wait_ms` timers for now though, I wasn't able to use counter the way I wanted.  Let me know if you have any other ideas.

Thanks